lang/image.bzl

```diff
@@ -263,11 +275,14 @@ image = struct(
         # The dependency whose runfiles we're appending.
         # If not specified, then the layer will be treated as the top layer,
         # and all remaining deps of "binary" will be added under runfiles.
```

Comment:
Do these comments need to be updated with our new change?

Reply:
I don't think so.  Using a transition should actually be the user's expectation.  It's the current behavior that is weird on Mac OSX.